sct.py

```diff
@@ -198,6 +199,9 @@ def provision_resources(backend, test_name: str, config: str):
 
     params = create_sct_configuration(test_name=test_name)
     test_config = get_test_config()
+    test_id = test_config.test_id()
+    if test_id is None or test_id == "None":
```

Comment:
isn't it better to?
```suggestion
    if test_id == "None" or not test_id:
```

Reply:
what is the difference? 
Also generally, according to Pycharm recommendations (based on pep8)  `None` values should use `is` for comparison with `None`

Reply:
there should be no case like that, but your'e right we should fail too for it. I'll fix it in some other PR.